Modules/DevTools/TestFramework/TestRunner/src/TestSuiteMgt.Codeunit.al

```diff
@@ -250,34 +250,73 @@ codeunit 130456 "Test Suite Mgt."
     procedure GetTestMethods(var ALTestSuite: Record "AL Test Suite"; var AllObjWithCaption: Record AllObjWithCaption)
     var
         TestLineNo: Integer;
+        DialogContent: Label '-- Get Test Methods --\#1#\#2#';
```

Comment:
Minor - Getting Test Methods

Reply:
OK, I will do this change in all new DialogContent Labels, including the change from Switching to Inverting as @JesperSchulz has proposed